src/lib/molecules/AdvancedSearch/SearchResultRow.js

```diff
@@ -70,19 +70,20 @@ function SearchResultRow({ element }) {
                 })}
             >
                 {actions?.map((action, index) => (
-                    <Button
-                        key={index}
-                        tabIndex={0}
-                        size="medium"
-                        icon={action.icon}
-                        onBlur={handleBlur}
-                        appearance="minimal"
-                        onFocus={handleFocus}
-                        cornerRadius="smooth"
-                        onClick={() => action?.onClick(element)}
-                        className="searchResultRow__action"
-                        ariaLabel={action.name || action.icon}
-                    />
+                    <Tooltip key={index} text={action?.description} isVisible={!!action?.description}>
+                        <Button
+                            tabIndex={0}
+                            size="medium"
+                            icon={action.icon}
+                            onBlur={handleBlur}
+                            appearance="minimal"
+                            onFocus={handleFocus}
```

Comment:
Add logic in the `handleFocus` function to handle the `Tooltip` visibility state.